src/endless/EndlessUsbTool.cpp

```diff
@@ -195,31 +195,40 @@ void CEndlessUsbToolApp::InitLogging()
 	fileName.Replace(L" ", L"");
 	fileName += s;
 	fileName += L".log";
-	if (CEndlessUsbToolDlg::IsUninstaller()) {
-		fileName = TempFilePath(fileName);
-	} else {
-		fileName = GET_LOCAL_PATH(fileName);
-	}
-
-	try {
-		BOOL result = m_logFile.Open(fileName, CFile::modeWrite | CFile::typeUnicode | CFile::shareDenyWrite | CFile::modeCreate | CFile::osWriteThrough);
-		if (!result) {
-			m_enableLogDebugging = false;
-		}
 
-		//uprintf("Log original date time %ls\n", s);
-		//uprintf("Application version: %s\n", RELEASE_VER_STR);
-		//uprintf("Windows version: %s\n", WindowsVersionStr);
-		//uprintf("Windows version number: 0x%X\n", nWindowsVersion);
-		//uprintf("-----------------------------------\n", s);
+	BOOL result = false;
+	// Try to write log alongside ourself, provided we're not in C:\endless
+	// which is unwriteable and about to be deleted(!)
+	if (!CEndlessUsbToolDlg::IsUninstaller()) {
```

Comment:
This might be one for another branch (it was like this before) but might we want to actually write the logfile in the current dir in the uninstaller case as well, provided we're not running from <sysdrive>:\endless?